### PR TITLE
Updated readme setup instructions

### PR DESCRIPTION
**Pull request by HeyImMatt**
- Added wheel install command
- Added step to follow on first runthrough when the virtual env will already be active.

### CODE REVIEW

**Comment by HeyImMatt** [comment]
Added the couple steps that worked for me. Pretty new to environment setup so feel free to suggest edits if I've stated anything incorrectly!

File: README.md [hunk 1]
```diff
@@ -11,27 +11,37 @@ NOTE: Database is SQLite3 via SQLAlchemy
 ### To Start Server
 
 1. Clone the repo (`git clone https://github.com/codeforpdx/dwellinglybackend.git`)
+
 2. Install Python ( https://realpython.com/installing-python/ )
-3. Install virtualenv onto system (if not already installed)
+
+3. Install Python Wheels
+
+    - `pip install wheel`
```
Comment (Chris-Boe) [inline README.md:19]: What is `wheel` used for? I don't think we've needed this before

**Comment by wittejm** [comment]
@Chris-Boe Related question about the existing project setup: why does it rely on both `virtualenv` and `pipenv`? I got the flask server running with only pipenv.

**Comment by KentShikama** [comment]
@HeyImMatt Could you please see if the setup instructions at https://github.com/codeforpdx/dwellinglybackend/pull/32 works?

**Comment by Chris-Boe** [comment]
> @Chris-Boe Related question about the existing project setup: why does it rely on both `virtualenv` and `pipenv`? I got the flask server running with only pipenv.

Not sure! I'm not too familiar with the setup of Python projects so I'd have to ask @htharker42

**Comment by Chris-Boe** [comment]
Hey Matt, if you would still like to contribute to the Readme, please add either comments of what you'd like to add to #38 ! Thanks

**Pull request closed without merging**